lldb/source/Breakpoint/BreakpointLocation.cpp

```diff
@@ -524,6 +524,9 @@ void BreakpointLocation::GetDescription(Stream *s,
           s->EOL();
           s->Indent("function = ");
           s->PutCString(sc.function->GetName().AsCString("<unknown>"));
+          s->EOL();
+          s->Indent("mangled function = ");
```

Comment:
Do we want to emit "mangled  function" for C which doesn't have mangled names?  It might be nicer to check whether the GetMangledName returns a mangled name, and only print it if it does?  Having:

function = main
mangled function = main

looks a little silly.

Reply:
That's doable, I'll have a look. (in the C case, we print "unknown" with the current impl, which is arguably more silly)

Reply:
I actually don't feel super strongly about this one, however.

Reply:
Yeah, we really shouldn't print "unknown".  You could make the fallback "", since there can't be a mangled name that's that, and then if you get back "" don't print in.